src/utils/useUpdateEffect/useUpdateEffect.ts

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+// useUpdateEffect is a useEffect that doesn't fire on initial render
+// for detecting when values change
+// https://stackoverflow.com/a/58217148
+
+const useIsMounted = function useIsMounted() {
+    const isMounted = React.useRef(false);
+
+    React.useEffect(function setIsMounted() {
+        isMounted.current = true;
+
+        return function cleanupSetIsMounted() {
```

Comment:
Again, eslint is not happy about the named function.

Reply:
annoying that the local `yarn lint` apparently isn't the linter the runners are using

Reply:
Very. Going to have to look into that.